app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,75 @@
+class TweetController < ApplicationController
+	get "/tweets" do 
+	  	if !!session[:id]
+	  		erb :"tweets/tweets"
+	  	else
+	  		redirect "/login"
+	  	end
+  	end
+
+  	get "/tweets/new" do 
+  		if !!session[:id]
+  			erb :"tweets/create_tweet"
+  		else
+  			redirect "/login"
+  		end
+  	end
+
+  	post "/tweets" do 
+  		user = User.all.find(session["id"])
+		if params[:content] == ""
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.